spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -333,6 +335,32 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
         printv('  Maximum axial damage ratio: ' + str(np.round(maximum_axial_damage_ratio, 2)),
                self.verbose, type='info')
 
+    def _compute_issue_bridge_ratio(self, idx):
+        """
+        :param idx: int, index of the lesion
+        Compute tissue bridge ratios:
+        - ventral_bridge_ratio: ventral midsagittal tissue bridge width divided by total midsagittal tissue bridge width
+        - dorsal_bridge_ratio: dorsal midsagittal tissue bridge width divided by total midsagittal tissue bridge width
+        """
+        # Get the dorsal and ventral tissue bridges
+        dorsal_bridge_width = float(self.measure_pd.loc[idx, 'interpolated_dorsal_bridge_width [mm]'])
+        ventral_bridge_width = float(self.measure_pd.loc[idx, 'interpolated_ventral_bridge_width [mm]'])
+
+        # Compute the total tissue bridge width
+        total_bridge_width = dorsal_bridge_width + ventral_bridge_width
+
+        # Compute the dorsal and ventral bridge ratios
+        dorsal_bridge_ratio = dorsal_bridge_width / total_bridge_width * 100 if total_bridge_width > 0 else 0
+        ventral_bridge_ratio = ventral_bridge_width / total_bridge_width * 100 if total_bridge_width > 0 else 0
```

Comment:
nit: I think maybe we want `0.0` instead of `0` just to keep the float type consistent?

Also, these one-liners are getting a little long, so it might help with readability to split them up:

```suggestion
    if total_bridge_width > 0:
        dorsal_bridge_ratio = (dorsal_bridge_width / total_bridge_width) * 100
        ventral_bridge_ratio = (ventral_bridge_width / total_bridge_width) * 100
    else:
        dorsal_bridge_ratio = 0.0
        ventral_bridge_ratio = 0.0
```

Reply:
sounds good! accepting the suggestion